pkg/k8shandler/services.go

```diff
@@ -16,24 +16,43 @@ import (
 func CreateOrUpdateServices(dpl *v1alpha1.Elasticsearch) error {
 	elasticsearchClusterSvcName := fmt.Sprintf("%s-%s", dpl.Name, "cluster")
 	elasticsearchRestSvcName := dpl.Name
+	metricsSvcName := fmt.Sprintf("%s-%s", dpl.Name, "metrics")
 	owner := asOwner(dpl)
 
 	labelsWithDefault := appendDefaultLabel(dpl.Name, dpl.Labels)
 
-	err := createOrUpdateService(elasticsearchClusterSvcName, dpl.Namespace, dpl.Name, "cluster", 9300, selectorForES("es-node-master", dpl.Name), labelsWithDefault, true, owner)
-	if err != nil {
+	if err := createOrUpdateService(elasticsearchClusterSvcName, dpl.Namespace, dpl.Name,
```

Comment:
purely cosmetic, can we update this function call to have an arg a line?
```
if err := createOrUpdateService(
  elasticsearchClusterSvcName,
  dpl.Namespace,
  ...
  owner); err != nil {
```

Reply:
Doesn't it look even worse then?
```
	if err := createOrUpdateService(elasticsearchClusterSvcName,
		dpl.Namespace,
		dpl.Name,
		"cluster",
		9300,
		selectorForES("es-node-master", dpl.Name),
		map[string]string{},
		labelsWithDefault,
		true,
		owner); err != nil {
		return fmt.Errorf("Failure creating service %v", err)
	}
```

Reply:
I don't know, but we should follow the go fmt/golang style guidelines, however ugly they may be.